### PR TITLE
chore: added default terms as title that would be shown when hovered

### PR DESCRIPTION
**Pull request by sunguru98**
The places where custom terms exist now have an option to view the default name by just hovering it for a while.

### CODE REVIEW

File: src/pages/Proposals.jsx [hunk 1]
```diff
@@ -28,6 +28,7 @@ const Proposals = React.memo(({
       as={RouterLink}
       to={`/dao/${daochain}/${daoid}/proposals/new`}
       rightIcon={<RiAddFill />}
+      title={customTerms ? 'proposal' : null}
```
Comment (scottrepreneur) [inline src/pages/Proposals.jsx]: looks good. should these be capital case to feel like proper nouns?

Comment (sunguru98) [reply src/pages/Proposals.jsx]: Oh yeah. Totally forgot about that .. Changing it now :)

Comment (scottrepreneur) [reply src/pages/Proposals.jsx]: I think if we want to do it on buttons/links we should handle the whole button or link's text.

Comment (scottrepreneur) [reply src/pages/Proposals.jsx]: I think if we want to do it on buttons/links we should handle the whole button or link's text.

let's get this to custom terms and then handle its translation.
Screen Shot 2021-05-05 at 11 03 28 AM

also 'member info' on dao overview
Screen Shot 2021-05-05 at 11 26 48 AM

member snapshot chart also
Screen Shot 2021-05-05 at 11 53 50 AM

**Comment by scottrepreneur** [comment]
I still couldn't find the right TextBox to get the one on DAO Overview, but this is good otherwise and we can clean up that one later. Worth a quick look if you can find it.

**Pull request merged**